initializers/server/index.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-console */
+const webpack = require('webpack');
+const webpackDevServer = require('webpack-dev-server');
+
+const config = require('../../webpack.config.js');
+
+const host = '10.0.2.15';
+const port = 3001;
+
+new webpackDevServer(webpack(config), {
+  contentBase: './dist',
+  // hot: true,
```

Comment:
bundle.js:69841 [HMR] The following modules couldn't be hot updated: (They would need a full reload!)
./node_modules/webpack/hot/log-apply-result.js.module.exports @ bundle.js:69841
(anonymous) @ bundle.js:69904
bundle.js:69843 [HMR]  - ./src/components/ui/BlogItem.jsx